### PR TITLE
Allow all source files to be filtered

### PR DESCRIPTION
**Pull request by baden03**
Allows a single filter to override any source file as mentioned in issue #77
Now, to overwrite the default profile.php file all that would need to be done is something like:

```
function my_tabs( $files, $sources ){
		$path = plugin_dir_path( __FILE__ );
		$files[1] = $path . 'sections/profile.php';
		$files[20]  = $path . 'metaboxes/profile-name.php';
		return $files;
}
add_filter( 'wp_user_profiles_core_files', 'my_tabs', 10, 2 );
```

### CODE REVIEW

**Comment by baden03** [comment]
Just dropped in to see [what condition my condition was in](https://www.youtube.com/watch?v=AULOC--qUOI)

**Comment by baden03** [comment]
Closing this pull request as it has been open for over a year with no response.

File: wp-user-profiles/includes/sections.php [hunk 1]
```diff
@@ -89,7 +89,7 @@ function wp_user_profiles_register_other_section() {
 }
 
 /**
- * Register the "Options" section
+ * Register the "Persissions" section
```
Comment (lkraav) [inline wp-user-profiles/includes/sections.php:92]: "Permissions" typo?

Comment (JJJ) [reply wp-user-profiles/includes/sections.php:92]: Permissions schmerschmissions...

Thank you 👍

Comment (JJJ) [reply wp-user-profiles/includes/sections.php:92]: Merged to master in b061b88132b35c3f27a86370198bc1ac0d79b4dd

**Pull request merged**